DependencyInjection/Compiler/MiddlewaresPass.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Doctrine\Bundle\DoctrineBundle\Middleware\ConnectionNameAwareInterface;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+use function array_keys;
+use function is_subclass_of;
+use function sprintf;
+
+final class MiddlewaresPass implements CompilerPassInterface
+{
+    /** @var string  */
+    private $connectionDefsParam;
+
+    /** @var string  */
+    private $middlewareTag;
+
+    public function __construct(
+        string $connectionDefsParam = 'doctrine.connections',
+        string $middlewareTag = 'doctrine.middleware'
```

Comment:
Why would this be configurable here if we hardcode it in DoctrineExtension?

Reply:
Not needed here indeed